Modix.Data/Models/Moderation/InfractionSearchCriteria.cs

```diff
@@ -57,18 +75,67 @@ public class InfractionSearchCriteria
         /// or non-null, (or both).
         /// </summary>
         public bool? IsDeleted { get; set; }
+
+        /// <summary>
+        /// Defines the searchable properties of an <see cref="InfractionSummary"/>.
+        /// </summary>
+        public static readonly ImmutableArray<string> SearchablePropertyNames = ImmutableArray.Create
```

Comment:
O_O

Was this some kinda temporary experiment? I can't find this and `SetPropertyValue` being used anywhere in this PR.